src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java

```diff
@@ -347,4 +349,52 @@ public static void populateDynamicListOffices(CaseData caseData, String caseType
                 : new ArrayList<>();
     }
 
+    public static CaseData populateDynamicRespondentRepresentativeNames(CaseData caseData) {
```

Comment:
Why not put these in a separate class? This Helper class is awful. You could create something like RespondentRepresentativeHelper class instead.